packages/chwp-deb/chwp-deb.pacscript

```diff
@@ -0,0 +1,12 @@
+name="chwp-deb"
+gives="chwp"
+breaks="${gives}"
+replace="${gives}"
+repology=("project: ${gives}")
+pkgver() {
+    git ls-remote --tags "https://github.com/rouhim/chwp" | cut -d/ -f3- | tail -n1 | cut -c 2-
+}
+version="$(pkgver)"
```

Comment:
Remove this and use `version="1.119.0`. (https://github.com/RouHim/chwp/releases/tag/v1.119.0)